6/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN cd /usr/src && \
     curl -o git-1.9.4.tar.gz https://www.kernel.org/pub/software/scm/git/git-1.9.4.tar.gz && \
     tar xzf git-1.9.4.tar.gz && cd git-1.9.4 && \
     make prefix=/usr all && make prefix=/usr install && \
-    cd .. && rm -r git-1.9.4.tar.gz git-1.9.4
+    cd .. && rm -rf git-1.9.4.tar.gz git-1.9.4
```

Comment:
This is so you don't get prompted for every file delete when you run these steps interactively.